strapi/strapi
# [DEITS] Add archive command option

### What does it do?

Adds the --archive CLI option that defaults to true, which adds the files to a tar archive instead of outputting them into a folder structure on the filesystem.

### Why is it needed?

So archive option can be toggled by the CLI

### How to test it?

Run `strapi export --archive false` or `strapi export --archive true` and the option should be passed through to the destination provider
